src/IdeHelperServiceProvider.php

```diff
@@ -35,7 +34,7 @@ class IdeHelperServiceProvider extends ServiceProvider implements DeferrableProv
      */
     public function boot()
     {
-        if ($this->app['config']->get('ide-helper.post_migrate', [])) {
+        if (!$this->app->runningUnitTests() && $this->app['config']->get('ide-helper.post_migrate', [])) {
```

Comment:
Will this work with Lumen? 😬

Reply:
It should: https://github.com/laravel/lumen-framework/blob/8.x/src/Application.php#L896

Reply:
Thank you, LGTM!

Could you please resolve the merge conflicts? Don't see any blockerS!